relayer/pkg/chainlink/ocr2/transmissions_cache.go

```diff
@@ -20,7 +20,6 @@ var _ median.MedianContract = (*transmissionsCache)(nil)
 type transmissionsCache struct {
 	transmissionDetails TransmissionDetails
 	tdLock              sync.RWMutex
-	tdTime              time.Time
```

Comment:
Interesting that this wasn't used...there should be a TTL check here similar to the contract_cache implementation